include/fcl/BV/RSS.h

```diff
@@ -38,10 +38,10 @@
 #ifndef FCL_RSS_H
 #define FCL_RSS_H
 
-
+#include <math.h>
```

Comment:
nitpick: I would prefer to use `<cmath>` since this is a C++ standard library header while `<math.h>` is a header defined by the C standard library.